File: Pemrograman 1/MODUL 6/Bahasa Python/PRAK605-2410817110002-Akhmad_Daffa_Azzikri.py
n = int(input())

print("Matriks A")
matriks_a = []
for i in range(n):
    baris = list(map(int, input().split()))
    matriks_a.append(baris)

print("Matriks B")
matriks_b = []
for i in range(n):
    baris = list(map(int, input().split()))
    matriks_b.append(baris)

print(f"Matriks {n}x{n}")
hasil = [[0 for _ in range(n)] for _ in range(n)]

for i in range(n):
    for j in range(n):
        for k in range(n):
            hasil[i][j] += matriks_a[i][k] * matriks_b[k][j]

for baris in hasil:
    print(' '.join(map(str, baris)))

File: Pemrograman 1/MODUL 6/Bahasa C/PRAK605-2410817110002-Akhmad_Daffa_Azzikri.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);

    int matriks_a[n][n], matriks_b[n][n], hasil[n][n];

    printf("Matriks A\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            scanf("%d", &matriks_a[i][j]);
        }
    }

    printf("Matriks B\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            scanf("%d", &matriks_b[i][j]);
        }
    }

   printf("Matriks %dX%d \n", n, n); 

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            hasil[i][j] = 0;
            for (int k = 0; k < n; k++) {
                hasil[i][j] += matriks_a[i][k] * matriks_b[k][j];
            }
        }
    }

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            printf("%d ", hasil[i][j]);
        }
        printf("\n");
    }

    return 0;
}